### PR TITLE
Add cluster ids to webhook requests

### PR DESCRIPTION
**Pull request by yonkoma**
Builds on changes from #5818

### CODE REVIEW

File: applications/webhooks/src/webhooks_util.erl [hunk 1]
```diff
@@ -181,11 +181,12 @@ maybe_fire_foldl(Key, Value, {_ShouldFire, JObj}) ->
 fire_hook(JObj, #webhook{custom_data = 'undefined'
                         } = Hook) ->
     EventId = kz_binary:rand_hex(5),
-    do_fire(Hook, EventId, JObj);
+    NewJObj = kz_json:insert_value(<<"cluster_id">>, list_to_binary(get_cluster_id()), JObj),
```
Comment (lazedo) [inline applications/webhooks/src/webhooks_util.erl]: remove `list_to_binary`

File: applications/webhooks/src/webhooks_util.erl [hunk 2]
```diff
@@ -757,3 +758,13 @@ fetch_available_events() ->
             kz_cache:store_local(?CACHE_NAME, ?AVAILABLE_EVENT_KEY, Events, CacheProps),
             Events
     end.
+
+-spec get_cluster_id() -> nonempty_string().
```
Comment (lazedo) [inline applications/webhooks/src/webhooks_util.erl]: return kz_term:ne_binary()

File: applications/webhooks/src/webhooks_util.erl [hunk 3]
```diff
@@ -757,3 +758,13 @@ fetch_available_events() ->
...
+get_cluster_id() ->
+    case kapps_config:get_string(?MOD_CONFIG_CLUSTER, <<"cluster_id">>) of
```
Comment (lazedo) [inline applications/webhooks/src/webhooks_util.erl]: use get_ne_binary_value

File: applications/webhooks/src/webhooks_util.erl [hunk 4]
```diff
@@ -757,3 +758,13 @@ fetch_available_events() ->
...
+        'undefined' ->
+            ClusterId = kz_binary:rand_hex(16),
+            {'ok', _JObj} = kapps_config:set_default(?MOD_CONFIG_CLUSTER, <<"cluster_id">>, ClusterId),
+            kz_term:to_list(ClusterId);
```
Comment (lazedo) [inline applications/webhooks/src/webhooks_util.erl]: return `ClusterId`

File: applications/webhooks/src/webhooks_util.erl [hunk 5]
```diff
@@ -181,11 +181,12 @@ maybe_fire_foldl(Key, Value, {_ShouldFire, JObj}) ->
...
     EventId = kz_binary:rand_hex(5),
-    do_fire(Hook, EventId, JObj);
+    NewJObj = kz_json:insert_value(<<"cluster_id">>, kzd_cluster:id(), JObj),
+    do_fire(Hook, EventId, NewJObj);
 fire_hook(JObj, #webhook{custom_data = CustomData
                         } = Hook) ->
-    EventId = kz_binary:rand_hex(5),
-    do_fire(Hook, EventId, kz_json:merge_jobjs(CustomData, JObj)).
+    NewJObj = kz_json:merge_jobjs(CustomData, JObj),
+    fire_hook(NewJObj, Hook#webhook{custom_data = 'undefined'}).
```
Comment (lazedo) [inline applications/webhooks/src/webhooks_util.erl]: why this change ?

File: core/kazoo_provisioner/src/provisioner_v5.hrl [hunk 6]
```diff
@@ -4,6 +4,7 @@
 -include_lib("kazoo_stdlib/include/kz_databases.hrl").
 
 -define(MOD_CONFIG_CAT, <<"provisioner">>).
+-define(MOD_CONFIG_CLUSTER, <<"cluster">>).
```
Comment (lazedo) [inline core/kazoo_provisioner/src/provisioner_v5.hrl]: not needed

File: applications/crossbar/src/modules/phonebook.erl [hunk 7]
```diff
@@ -17,6 +17,7 @@
 -include_lib("kazoo_number_manager/include/knm_port_request.hrl").
 
 -define(MOD_CONFIG_CAT, <<"crossbar.phonebook">>).
+-define(MOD_CONFIG_CLUSTER, <<"cluster">>).
```
[the remark this answers is not in the document]
Comment (yonkoma) [reply applications/crossbar/src/modules/phonebook.erl]: oh, this must've slipped in by accident

File: applications/webhooks/src/webhooks_util.erl (hunk 5 above)
Comment (lazedo) [reply applications/webhooks/src/webhooks_util.erl]: nm

Comment (yonkoma) [reply applications/webhooks/src/webhooks_util.erl]: To make it all calls go through one point so that there aren't repeated calls for adding stuff to the request

Comment (lazedo) [reply applications/webhooks/src/webhooks_util.erl]: yep, noticed that after, `nm` => `never mind`
